system-helper/flatpak-system-helper.c

```diff
@@ -1536,7 +1672,9 @@ handle_revokefs_fd (FlatpakSystemHelper   *object,
   flatpak_system_helper_complete_revokefs_fd (object, invocation,
                                               fd_list, g_variant_new_handle (fd_index),
                                               new_pull->src_dir,
-                                              new_pull->mountpoint);
+                                              new_pull->mountpoint,
+                                              partial_pull_availablity,
```

Comment:
I don't think this parameter is needed: if the path is non-empty, then it is available. If not, it's not.